src/res/KM_Resource.pas

```diff
@@ -64,6 +64,8 @@   TKMResource = class
     property Tileset: TKMTileset read fTileset;
     property UnitDat: TKMUnitDatCollection read fUnitDat;
 
+    function GetHouseNotOccupiedMsgId(aHouseType: THouseType): Integer;
```

Comment:
It's better to rename "NotOccup" to "Unoccupied", so that the term used is consistent.